packages/react-dev-utils/README.md

```diff
@@ -170,7 +170,7 @@ module: {
 
 Captures JS and CSS asset sizes inside the passed `buildFolder`. Save the result value to compare it after the build.
 
-##### `printFileSizesAfterBuild(webpackStats: WebpackStats, previousFileSizes: OpaqueFileSizes)`
+##### `printFileSizesAfterBuild(webpackStats: WebpackStats, previousFileSizes: OpaqueFileSizes, buildFolder: string)`
```

Comment:
Yes sorry! It's a long week-end here in France (day off) but I do it right when I come back, tonight. 

Do you want me to add another phrase to describe the new arguments or I add this in the signature only?